splinepy/helpme/integrate.py

```diff
@@ -0,0 +1,125 @@
+import numpy as np
+
```

Comment:
```suggestion

from splinepy.utils.data import cartesian_product

```
If this doesn't cause any import issues, I'd prefer this